docs/content/en/docs/Architecture/container.md

```diff
@@ -51,22 +51,22 @@ $ docker pull quay.io/kairos/core-alpine:v1.1.0
 We can run it locally with docker as a container to inspect it, as it is runnable:
 
 ```
-$ docker run -ti --rm quay.io/kairos/core-alpine:v1.1.0
+$ docker run -ti --rm quay.io/kairos/core-alpine:v2.0.0
```

Comment:
I think this is wrong, we don't have core-alpine anymore, but core-alpine-ubuntu, etc. 

Maybe just use debian? should be the same I guess

Reply:
yup good idea, decided to go with a site variable for these, won't replace everywhere to avoid making the PR too big